sdk/clientcore/core/src/main/java/io/clientcore/core/utils/Base64Uri.java

```diff
@@ -77,10 +78,19 @@ public static Base64Uri encode(byte[] bytes) {
         if (bytes == null) {
             return new Base64Uri((String) null);
         } else {
-            return new Base64Uri(Base64Util.encodeURIWithoutPadding(bytes));
+            return new Base64Uri(encodeURIWithoutPadding(bytes));
         }
     }
 
+    /**
+     * Encodes a byte array to base64 URI format.
+     * @param src the byte array to encode
+     * @return the base64 URI encoded bytes
+     */
+    private static byte[] encodeURIWithoutPadding(byte[] src) {
```

Comment:
Should we remove Base64Uri too in favor of [Base64](https://docs.oracle.com/javase/8/docs/api/java/util/Base64.html)?

Reply:
Will need to see where we use this today / in Track 2 libraries before we make a decision here.